docs/release_notes.md

```diff
@@ -8,6 +8,59 @@ We are tracking two types of major changes:
 
 If you are creating the release notes for a new version, please see the [template](#template-version-majorminorpatch). Release notes should capture all of the significant changes since the last numbered package release.
 
+# Version 1.1.0
+
+This version focuses on improving the clarity of the telemetry reported.
+
+## Users
+
+### User Improvements
+
+- ONNX files exported from PyTorch models now have a `torch_export_verified` key in their stats/report files that indicates whether the `torch.onnx.verification.find_mismatch()` API could find any issue with the exported ONNX file.
+- Stats and report CSV files split `stages_completed` into stage status and duration.
+- Build, benchmark, and stage status values in the stat and report files now use the same terminology values:
+
+```
+class FunctionStatus(enum.Enum):
+    # INCOMPLETE indicates stage/build/benchmark is either running or was killed;
+    # if you know the process ended then it was killed;
+    # if the process is still running, stage/build/benchmark is still running.
+    INCOMPLETE = "incomplete"
+    # NOT_STARTED applies to stages that didnt start because
+    # the build errored out or was killed prior to stage starting.
```

Comment:
Previous sentence sounds like the condition here is `didn't start + reason for not starting`. What we actually want to say is `didn't start`, which may be caused by `reason`.
```suggestion
    # NOT_STARTED applies to stages that didn't. This happens when
    # the build errors out or when the build is killed prior to stage starting.
```

Reply:
Incorporated into some more wordsmithing